automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala

```diff
@@ -299,8 +303,8 @@ class WorkspaceSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures
     "who has reader access to workspace" - {
 
       "should see launch analysis button disabled" in withWebDriver { implicit driver =>
-        val user = UserPool.chooseStudent
-        implicit val authToken: AuthToken = authTokenOwner
+        val user = Config.Users.owner
```

Comment:
This doesn't seem right since it's supposed to be a user with reader access

Reply:
Oh wow, I didn't even realize I had changed that...

Reply:
Hah. It happens!
Another win for the review process!

Reply:
Actually - it might be because I didn't rebase